Mathlib/Geometry/Manifold/ChartedSpace.lean

```diff
@@ -693,6 +694,24 @@ theorem ChartedSpace.locallyConnectedSpace [LocallyConnectedSpace H] : LocallyCo
   · rintro x s ⟨⟨-, -, hsconn⟩, hssubset⟩
     exact hsconn.isPreconnected.image _ ((e x).continuousOn_symm.mono hssubset)
 
+/-- If a topological space admits an atlas with locally path-connected charts, then the space itself
```

Comment:
How about "if a charted space `M` admits at atlas ... then `M` itself is locally connected"?

Reply:
The charted space structure *is* the atlas, right? So I think saying "a topological space" makes more sense. I did name the space `M` now though, and also fixed it to say locally path-connected instead of locally connected.

Reply:
Well, a charted space might have several atlasses, one of which is connected, the other of which is not. (For local path-connectedness, that's probably all the same, though.)
I think the current phrasing is fine, though.